api/client/builder/client.go

```diff
@@ -295,3 +307,12 @@ func non200Err(response *http.Response) error {
 		return errors.Wrap(ErrNotOK, errMessage.Message)
 	}
 }
+
+func covertEndPoint(ep string) network.Endpoint {
```

Comment:
This would be useful as a general helper in a common location, we do something like this in execution service as well. Another PR will be great for this